docs/ref/models/fields.txt

```diff
@@ -1292,6 +1292,38 @@ The possible values for :attr:`~ForeignKey.on_delete` are found in
     :exc:`~django.db.models.ProtectedError`, a subclass of
     :exc:`django.db.IntegrityError`.
 
+* .. attribute:: RESTRICT
+
```

Comment:
Add a `.. versionadded:: 1.11` annotation and a mention in docs/releases/1.11.txt. See our [patch review checklist](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#patch-review-checklist).